### PR TITLE
[SAJC] Introduce the SAJC (Application Job Catalogs) to AFF

### PR DESCRIPTION
**Pull request by albertmink**
https://jira.tools.sap/browse/ADTVEGA-1452

### CODE REVIEW

File: file-formats/sajc/examples/z_aff_example_sajc.sajc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "in non elit",
+    "originalLanguage": "fj",
+    "abapLanguageVersion": "standard"
```
Comment (albertmink) [inline file-formats/sajc/examples/z_aff_example_sajc.sajc.json:6]: @wurzka is this an error from the generation tool?

Comment (wurzka) [reply file-formats/sajc/examples/z_aff_example_sajc.sajc.json:6]: No, it should be removed here. The ST wouldn't write it to the JSON data file

File: file-formats/sajc/examples/z_aff_example_sajc.sajc.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "in non elit",
+    "originalLanguage": "fj",
```
Comment (albertmink) [inline file-formats/sajc/examples/z_aff_example_sajc.sajc.json:5]:
```suggestion
    "originalLanguage": "en"
```

File: file-formats/sajc/examples/z_aff_example_sajc.sajc.json (hunk 1 above)
Comment (albertmink) [inline file-formats/sajc/examples/z_aff_example_sajc.sajc.json:6]:
```suggestion
```

Comment (albertmink) [reply file-formats/sajc/examples/z_aff_example_sajc.sajc.json]: done in [641b254](https://github.com/SAP/abap-file-formats/pull/313/commits/641b25443b1c2dc68b995c20eb36868e92f3aeec)

File: file-formats/sajc/sajc-v1.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
```
Comment (wurzka) [inline file-formats/sajc/sajc-v1.json:42]: I tried to find out which different types of ABAP Language Version are possible for SAJC but only found some with "standard". 
@burcuka Do you know more about this or should we ask SAJC colleagues about the language version?

Comment (burcuka) [reply file-formats/sajc/sajc-v1.json:42]: I would expect these possible values but Im not sure... maybe we could ask the colleagues

Comment (burcuka) [reply file-formats/sajc/sajc-v1.json:42]: I asked the SAJC colleagues.. "standard" and "cloudDevelopment" will be supported as ABAP language versions

Comment (wurzka) [reply file-formats/sajc/sajc-v1.json:42]: Then we should either change zif_aff_types_v1=>ty_header_120 to use zif_aff_types_v1=>co_abap_language_version_cloud (and maybe rename it to for example ty_header_120_cloud) or SAJC has to define its own header in its interface

File: file-formats/sajc/type/zif_aff_sajc_v1.intf.abap [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      "! Description of the ABAP object
+      "! $required
+      description           TYPE c LENGTH 120,
+      "! <p class="shorttext">Original Language</p>
+      "! Original language of the development object.
+      "! $required
+      original_language     TYPE sy-langu,
+      "! $required
+      "! $values {@link zif_aff_types_v1.data:co_abap_language_version_cloud}
+      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version,
```
Comment (albertmink) [inline file-formats/sajc/type/zif_aff_sajc_v1.intf.abap:16]: @wurzka is the language version required, or it this bad copy&pasting of source code?

Comment (wurzka) [reply file-formats/sajc/type/zif_aff_sajc_v1.intf.abap:16]: Language version is not required. But it should have a default

**Review by wurzka — APPROVED** [review]

**Pull request merged**